actions/DisplayAction.php

```diff
@@ -188,19 +178,31 @@ public function execute()
                         );
 
                         $item->setTimestamp(time());
-                        $item->setContent(buildBridgeException($e, $bridge));
+
+                        $message = sprintf(
+                            'Uncaught Exception %s: %s at %s line %s',
+                            get_class($e),
+                            $e->getMessage(),
+                            trim_path_prefix($e->getFile()),
+                            $e->getLine()
+                        );
+
+                        $content = render_template('bridge-error.html.php', [
```

Comment:
Wait, does not `Configuration::getConfig('error', 'output') === 'feed'` mean that they want the errors as items in the selected feed format, rather than HTML?

Reply:
Yes that's right. `render_template()` renders without base template as opposed to `render()` which renders using a base template.

Reply:
And yes the prior behaviour is to render html into the item regardless of feed format.